sdk/search/ai-search/README.md

```diff
@@ -0,0 +1,302 @@
+# Azure Cognitive Search client library for JavaScript
+
+[Azure Cognitive Search](https://docs.microsoft.com/en-us/azure/search/) is a search-as-a-service cloud solution that gives developers APIs and tools for adding a rich search experience over private, heterogeneous content in web, mobile, and enterprise applications.
+
+Azure Cognitive Search is well suited for the following application scenarios:
+
+- Consolidation of heterogeneous content types into a private, single, searchable index. Queries are always over an index that you create and load with documents, and the index always resides in the cloud on your Azure Cognitive Search service. You can populate an index with streams of JSON documents from any source or platform. Alternatively, for content sourced on Azure, you can use an indexer to pull data into an index. Index definition and management/ownership is a key reason for using Azure Cognitive Search.
```

Comment:
This seems _really_ long for a short blurb/introduction. Is there any way to shrink this reasonably? I wouldn't want to take away meaning, I just find it intimidating to see that much text and it pushes the actual source/doc links way beneath the "fold".

On another note - there are some terms used here that we could at least call out as links. For instance, `cognitive skills` is probably worth linking out to since that's one of our service-specific terms, etc...

Reply:
I lifted this text from the product docs, agreed it is a bit too wordy. Will try to hack it down.